tests/integration/testdata/sync/template.yaml

```diff
@@ -0,0 +1,70 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
nit: Can we move infra sync testdata into a subfolder of testdata/sync? We probably will need more version of this for code syncs and other tests in the future.
`testdata/sync/base-infra-stack` or something similar.

Reply:
Yes I'm doing a new ordering of folder structures! Thanks!